tasmota/tasmota_support/support.ino

```diff
@@ -590,6 +590,33 @@ char* Trim(char* p) {
   return p;
 }
 
+size_t HexStringToBinaryString(char** source) {
```

Comment:
Please use `ToHex_P()` instead of adding a new implementation

Reply:
The ToHex_P function takes an array and converts it into a hex string. I'm doing the inverse of that.
It takes the binary value and maps it to one of the characters in `"static const char * hex PROGMEM = "0123456789ABCDEF";"`